### DISCUSSION TITLE
Disable "deprecated" JDT warnings

### DISCUSSION THREAD

**Discussion by Bananeweizen** [eclipse-tycho/tycho#1949] [Ideas]
I'd like to disable the deprecated warning in all JDT/PDE project settings of Tycho bundles/modules. That would lead to way less warnings (and the deprecated things cannot be fixed in almost all cases that I checked). Looks like this locally (with a filter applied instead) and would make it possible to actually recognize newly introduced warnings.
![grafik](https://user-images.githubusercontent.com/406876/211573100-af713a5b-36c0-4ee0-bc50-0494347eb839.png)

**Comment by laeubi** [comment]
DO you men with `@SuppressWarning`? Would be fine for me if you already have checked that nothing can be done...
